Elevel/Application/Features/QuestionCommands/GetQuestionListQuery.cs

```diff
@@ -0,0 +1,65 @@
+﻿using AutoMapper;
+using AutoMapper.QueryableExtensions;
+using Elevel.Application.Interfaces;
+using Elevel.Application.Pagination;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Elevel.Application.Features.QuestionCommands
+{
+    public class GetQuestionListQuery
+    {
+        public class Request : PagedQueryBase, IRequest<Response>
+        {
+            public Guid Id { get; set; }
+        }
+
+        public class Handler : IRequestHandler<Request, Response>
+        {
+            private readonly IApplicationDbContext _context;
+            private readonly IMapper _mapper;
+
+            public Handler(IApplicationDbContext context, IMapper mapper)
+            {
+                _context = context;
+                _mapper = mapper;
+            }
+
+            public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
+            {
+                var questions = _context.Questions.AsNoTracking()
+                    .Where(x => x.Id == request.Id);
```

Comment:
these list always will have 1 question

Reply:
Where -> Include, plus => x.Id instead of equality